haskell/hx-adl/helix-adl-tools.cabal

```diff
@@ -1,40 +1,37 @@
+cabal-version:       2.4
 name:                helix-adl-tools
 version:             0.1.0.0
 synopsis:            Helix specific ADL tools
--- description:
---homepage:
-license:             BSD3
+license:             BSD-3-Clause
 license-file:        LICENSE
 author:              Tim Docker
 maintainer:          timd@helixta.com.au
 copyright:           2017 Tim Docker
 category:            Database
 build-type:          Simple
-cabal-version:       >=1.10
+
 
 executable hx-adl-hs
   hs-source-dirs:      src
   main-is:             Main.hs
   default-language:    Haskell2010
-  build-depends:       base >= 4.7 && < 5
-                     , aeson >= 1.0.2.1 && < 1.5
+  build-depends:       base
+                     , aeson
                      , adl-runtime
-                     , adl-utils
-                     , adl-compiler-lib
                      , adl-compiler
                      , helix-adl-tools
-                     , base64-bytestring >= 1.0.0.1 && < 1.1
```

Comment:
Why the omission of a version range?

Reply:
Because they are unnecessary when building binaries.  When using stack the resolver locks down all the versions in any case. We updated to

lts-17.5 which bakes in the version of the compiler (ghc-8.10.4) and all of the baked in packages here:

https://www.stackage.org/lts-17.5

If building and publishing library code, intended to be used against different dependency versions, then you would want the range.